eva/udfs/emotion_detector.py

```diff
@@ -104,13 +103,13 @@ def __init__(self, threshold=0.85):
 
         # load model
         self.model = VGG("VGG19")
-        output_directory = os.path.join(EVA_DEFAULT_DIR, "udfs", "models")
+        output_directory = os.path.join(os.getcwd(), "models")
```

Comment:
We shouldn't need to change this. This should work across platforms

Reply:
Yes, made the change.